src/main/java/appeng/tile/storage/SkyChestTileEntity.java

```diff
@@ -118,22 +105,43 @@ private void onOpenOrClose() {
             this.world.addBlockEvent(this.pos, block, 1, this.numPlayersUsing);
             this.world.notifyNeighborsOfStateChange(this.pos, block);
             // FIXME: Uhm, we are we doing this?
-            this.world.notifyNeighborsOfStateChange(this.pos.down(), block);
+            // this.world.notifyNeighborsOfStateChange(this.pos.down(), block);
```

Comment:
Please don't leave commented out code. Just remove it (the FIXME comment too)

Reply:
Ah, missed this, thanks :)